### PR TITLE
feat(packages): turn package into also pod

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#325

### CODE REVIEW

File: MetriportSDK.podspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name             = 'MetriportSDK'
+  s.version          = '0.1.0'
+  s.summary          = 'A Swift Library for access to Apple Healthkit.'
+
+  s.homepage         = 'https://github.com/metriport/metriport-ios-sdk'
+  s.license          = { :type => 'MIT', :file => 'LICENSE.md' }
```
Comment (Orta21) [inline MetriportSDK.podspec:7]: What license do we use for this?

Comment (Orta21) [reply MetriportSDK.podspec:7]: Should i use same one in OSS?

File: MetriportSDK.podspec [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name             = 'MetriportSDK'
+  s.version          = '0.1.0'
```
Comment (Goncharo) [inline MetriportSDK.podspec]: Should be bump the version to 1.0.0?

File: MetriportSDK.podspec [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  s.summary          = 'A Swift Library for access to Apple Healthkit.'
```
Comment (Goncharo) [inline MetriportSDK.podspec]: "... for Metriport API and Apple Health integrations."

File: MetriportSDK.podspec (hunk 1 above)
Comment (Goncharo) [reply MetriportSDK.podspec:7]: Let's use MIT for this so there aren't limitations in cx implementations.

File: MetriportSDK.podspec [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  s.homepage         = 'https://github.com/metriport/metriport-ios-sdk'
+  s.license          = { :type => 'MIT', :file => 'LICENSE.md' }
+  s.author           = { 'Metriport' => 'engineering@metriport.com' }
```
Comment (Goncharo) [inline MetriportSDK.podspec]: contact@metriport.com

**Review by Goncharo — CHANGES_REQUESTED** [review]
Minor comments

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**